plugins/bc-theme-file/tests/TestCase/Form/ThemeFolderFormTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace BcThemeFile\Test\TestCase\Form;
+
+use BaserCore\TestSuite\BcTestCase;
+use BcThemeFile\Form\ThemeFolderForm;
+use Cake\Form\Schema;
+
+class ThemeFolderFormTest extends BcTestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
```

Comment:
ở đây a tạo lấy 1 đối tượng $this->ThemeFolderForm giống như mấy thằng test khác ý. @thangnnmd